### PR TITLE
Fix typographic errors in README.md

### PR DESCRIPTION
**Pull request by yuhr123**

### CODE REVIEW

**Comment by davies** [comment]
please rebase your branch on recent main branch.

**Comment by yuhr123** [comment]
> please rebase your branch on recent main branch.

All done!

File: README.md [hunk 1]
```diff
@@ -7,10 +7,19 @@
 </p>
 **JuiceFS** is a high-performance [POSIX](https://en.wikipedia.org/wiki/POSIX) file system released under GNU Affero General Public License v3.0. It is specially optimized for the cloud-native environment. Using the JuiceFS to store data, the data itself will be persisted in object storage (e.g. Amazon S3), and the metadata corresponding to the data can be persisted in various database engines such as Redis, MySQL, and SQLite according to the needs of the scene. 
 
+<<<<<<< HEAD
```
Comment (davies) [inline README.md:10]: There are still conflicts

**Comment by yuhr123** [comment]
Let me try to close this PR, delete the docs branch and make a new branch from main.

**Pull request closed without merging**